components/resources/brave_news_strings.grdp

```diff
@@ -13,7 +13,7 @@
     Turn on Brave News
   </message>
   <message name="IDS_BRAVE_NEWS_SEARCH_PLACEHOLDER_LABEL" desc="Label that displays in the searchbox when the user hasn't entered anything">
-    Search for news, site, topic or RSS feed
+    Search for news, site, topic, or RSS feed
```

Comment:
This has just made me focus on what this says and ponder why we are asking the user to search for "news"...

Reply:
Hahaha yeah, I didn't even really think about it. Matt has an update (and it should be `Search for site, topic, or RSS feed`)